_includes/release-notes/new-features.md

```diff
@@ -17,8 +17,6 @@ Refer to the [Context Toolbar]({{site.baseurl}}/ui-components/contexttoolbar/) d
 
 The Custom Menu items configurations options have changed in TinyMCE 5.0. The `addMenuItem` has a new configuration. A new method `addToggleMenuItem` has been added which is an explicit method for creating toggle menu items similar to the new special toolbar button methods.
 
-Refer to the [configuration]({{site.baseurl}}/ui-components/typesoftoolbarbuttons/#menubutton) documentation, for more information on `editor.menuitems`.
```

Comment:
This link refers to something else. We need to write DOC-112 before we can link anything here.

Reply:
Ok